documentation/src/cli.md

```diff
@@ -33,10 +33,24 @@ forest-cli --token eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJBbGxvdyI6WyJyZWFkIiwi
 ```
 
 When the Forest daemon is started, an admin token will be displayed (alternatively, use `--save-token <token>` flag to save it on disk).
+## Sending filecoin tokens (money) from your wallet.
+
+For sending filecoin token, forest daemon must be running. You can do so by running:
+
+`forest --chain calibnet --target-peer-count 50 --encrypt-keystore false`
+
+Next, to send filecoin token (money) to a wallet address:
+
+`forest-cli --token <admin_token> send <wallet-address> <amount in attoFIL>`
+
+where 1 attoFIL = 10e−18 FIL.
```

Comment:
It's not a pretty representation and I believe it's not accurate - 10e-18 is 10 attoFIL, no? Let's utilize some markdown capabilities:
```suggestion
where 1 attoFIL = $10^{−18}$ FIL.
```